packages/auth/api-extractor.json

```diff
@@ -1,6 +1,6 @@
 {
     "extends": "../../config/api-extractor.json",
-    "mainEntryPointFilePath": "<projectFolder>/dist/esm5/index.d.ts",
+    "mainEntryPointFilePath": "<projectFolder>/dist/esm5/index.doc.d.ts",
```

Comment:
I don't have enough context around why we had index.d.ts as the main entry point instead of index.doc.d.ts - maybe @sam-gc to weigh in?

Reply:
Sorry, this is an artifact of the doc generation process and shouldn't be checked in. Removing it.